libs/sdk/eslint-config/package.json

```diff
@@ -34,6 +34,6 @@
     "comment-json": "4.2.3",
     "fs-extra": "11.2.0",
     "jsonc-parser": "3.2.0",
-    "package-json": "8.1.1"
+    "package-json": "7.0.0"
```

Comment:
v8 introduced exclusive support for ESM, but `@skyux-sdk/eslint-config` outputs to `common-js` and throws an error so we'll need to stick with v7 for now.
```
require() of ES Module not supported.
```